internal/cmd/plugin/install/generate.go

```diff
@@ -98,7 +102,7 @@ func newGenerateCmd() *cobra.Command {
 	cmd.Flags().Int32Var(
 		&replicas,
 		"replicas",
-		0,
+		1,
```

Comment:
```suggestion
		0,
```

I would not change this in this patch - assuming it makes sense to change it.